hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java

```diff
@@ -422,6 +425,32 @@ final Stream<DynamicTest> consequentAirdrops() {
                             // assert the account balance
                             getAccountBalance(receiver).hasTokenBalance(FUNGIBLE_TOKEN, 10));
         }
+
+        @HapiTest
+        @DisplayName("that is alias with 0 free maxAutoAssociations")
+        final Stream<DynamicTest> autoCreateWithNftFallBackFeeFails() {
```

Comment:
Test name is not right for this test (copy/paste error/)

Also, this seems to be a success test.  The linked issue is ambiguous: is it a success test (as written here) or a failure test (the failure would be: no more free association slots and existing slot used by a _different_ token) - I didn't go and check the test plan (actually I'm not sure where it is).

Reply:
renamed